### ISSUE TITLE
precision on og-usa output page

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#136]
What do you think about providing tenth of a percent precision rather than hundredth of a percent precision on the OG-USA output?

**Comment by rickecon** [comment]
I think a hundredth-of-a-percent precision is best. I think the source of @MattHJensen 's question actually reveals a bug. I don't believe that all the output from the recent runs really has all zeros in the hundredth-of-a-percent position. We should look into this. @talumbau @jdebacker

**Comment by MattHJensen** [comment]
> I think the source of @MattHJensen<https://github.com/MattHJensen> 's question actually reveals a bug. I don't believe that all the output from the recent runs really has all zeros in the hundredth-of-a-percent position.

My suggestion is actually unrelated to this bug.

I was just thinking that all the decimal points oversell the precision of these models, even when the models are working perfectly. I can see that hundredths is probably best in an academic paper where the goal is to show what the model is doing, but this is a different setting where the goal is to offer a prediction of economic outcomes stemming from some policy changes.

On Jan 22, 2016, at 1:40 AM, Richard Evans <notifications@github.com<mailto:notifications@github.com>> wrote:


I think a hundredth-of-a-percent precision is best. I think the source of @MattHJensen<https://github.com/MattHJensen> 's question actually reveals a bug. I don't believe that all the output from the recent runs really has all zeros in the hundredth-of-a-percent position. We should look into this. @talumbau<https://github.com/talumbau> @jdebacker<https://github.com/jdebacker>

**Comment by rickecon** [comment]
@MattHJensen makes a good point that I agree with. Let’s go with tenth-of-a-percent reporting on TaxBrain.

**Comment by talumbau** [comment]
@MattHJensen @rickecon yep this is a bug, in the sense that it is rounding to the nearest tenth of a percent and then displaying hundredths of a percent. I will fix the formatting so that it only displays to a tenth of a percent.
